src/App.js

```diff
@@ -39,13 +55,18 @@ class App extends Component {
               </Navbar.Brand>
               <Navbar.Toggle />
             </Navbar.Header>
+            <Button onClick={this.toggleEditMode}>Edit</Button>
```

Comment:
Is there a react bootstrap component that behaves more like a toggle/switch? With a button, it's not clear what the state is (whether you're in edit mode or not)